code/modules/surgery/organs/autoimplanter.dm

```diff
@@ -63,3 +63,33 @@
 	desc = "A single use autoimplanter that contains a medical heads-up display augment. A screwdriver can be used to remove it, but implants can't be placed back in."
 	storedorgan = new/obj/item/organ/cyberimp/eyes/hud/medical()
 	uses = 1
+
+/obj/item/device/autoimplanter/esword
+	name = "Energy Blade Autoimplanter"
+	desc = "A single use autoimplanter that contains an energy blade implant, removable by screwdriver. However, once you remove it, you can no longer put it back in."
+	storedorgan = new/obj/item/organ/cyberimp/arm/esword()
+	uses = 1
+
+/obj/item/device/autoimplanter/baton
+	name = "Stun-Arm Autoimplanter"
+	desc = "A single use autoimplanter that contains a stun-arm implant, removable by screwdriver. However, once you remove it, you can no longer put it back in."
+	storedorgan = new/obj/item/organ/cyberimp/arm/baton()
+	uses = 1
+
+/obj/item/device/autoimplanter/medbeam
```

Comment:
See comment on the uplink datum.
You only really need to make the autoimplanter children for the esword and flash implants as those are the only ones that are in the traitor uplink.